identity-service/src/notifications/notificationProcessing.js

```diff
@@ -393,3 +430,19 @@ async function _processCreateNotifications (notif, blocknumber, timestamp, tx) {
 module.exports = {
   indexNotifications
 }
+
+async function _shouldNotifyUser (notificationTarget, prop, tx) {
```

Comment:
When calling this method, the second param prop could be an enum instead of passing in a string just so that it's a little safer, but doesn't need to be changed.